conans/test/integration/hooks/hook_test.py

```diff
@@ -11,33 +11,53 @@
 
 def pre_export(conanfile):
     conanfile.output.info("Hello")
+    # TODO: To have the export_folder here needs a bit more deep refactoring
```

Comment:
How bad is this? how is the pre_export supposed to work? it might need also the export_folder, right?

Reply:
``pre_export`` hook probably shouldn't have access to the export folders? If it happens before the actual export, those folders are not even created. Maybe ``pre_export`` is intended to operate mostly in the user space?

Reply:
why not? I might need to export some files from hook before conan exports from recipe

Reply:
some ConanCenter example? And some example that cannot be done in the post_hook?